packages/client/src/components/captcha.vue

```diff
@@ -27,8 +27,7 @@ type CaptchaContainer = {
 };
 
 declare global {
-	interface Window extends CaptchaContainer {
-	}
+	type Window = CaptchaContainer
```

Comment:
As I said these changes were all automatical with `eslint --fix`. This is the relevant lint annotation for this change:

```
/home/runner/work/misskey/misskey/packages/client/src/components/captcha.vue
Error:   30:12  error  An interface declaring no members is equivalent to its supertype  @typescript-eslint/no-empty-interface
```

description relevant to the lint: https://typescript-eslint.io/rules/no-empty-interface/

Reply:
I saw, but the description in the link you provided;

> An empty interface is equivalent to its supertype. If the interface does not implement a supertype, then the interface is equivalent to an empty object (`{}`). In both cases it can be omitted.

is not strictly accurate.

https://www.typescriptlang.org/play?#code/JYOwLgpgTgZghgYwgAgEIE8CS5rycgbwChlTkBnAewFsIAuZAchmAgBsATcxogXyKJh0ABxQBRAB6QQ5YJRDIAvIRJkA2gH1koJjEqVGyAD5MARnCiMAugxABXaqeh8BoSLEQoM2d3hQQpCBAuZElpWXkVMmQAehjkMAALYHJtVIRKKCgIBDAXAUERL3QAQTZgOFTlYmjKJOgGZlZObnyhUTRS8sqlUMCZOQU4hOTUlOQQOuQMrJy8oA

Reply:
typescript-eslint/typescript-eslint#1640

Reply:
I will configure the rule with `allowSingleExtends: true` as suggested in that issue so the lint does not show up as wrong any more.